src/az-databricks/README.md

```diff
@@ -408,6 +408,18 @@ If you don't want to allow all the trusted services to be able to access Azure K
 
 A different approach is to use REST API calls to retrieve the Azure Key Vault secrets. And this will work if you have the private endpoints created for Azure Key Vault from Databricks VNet. But it has other downside that the retrieved password won't be redacted and thus this approach needs to be applied carefully.
 
+### **DBFS root** storage account has public access enabled
+
+The **DBFS root** is the default storage location for an Azure Databricks workspace, provisioned as part of workspace creation in the cloud account containing the Azure Databricks workspace. It contains a number of special locations that serve as defaults for various actions performed by users in the workspace, the details can be checked [here](https://learn.microsoft.com/azure/databricks/dbfs/root-locations). The root Blob storage contains workspace’s DBFS root.
```

Comment:
For the last sentence, "The root Blob storage . . . ", is this a blob storage container which contains the workspace's DBFS root?

Reply:
yes, that's correct. Updated